### PR TITLE
Finish_Mobile_First

### PR DESCRIPTION
**Pull request by manzitresor**
Finishing Mobile Version of Portfolio Project Using HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
             <li><a href="#"><img src="./images/Vector (3).png"></a></li> 
           </ul>
     </section>
+    <section class="grid-container">
+       <div class="grid-1">
+         <img src="./images/Snapshoot Portfolio.png" alt="project1" class="project1">
+         <h1 class="ton">Tonic</h1>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 2]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+                <ul class="categories">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JAVASCRIPT</li>
+                </ul>
+            <button class="see-project">See Project</button>
+       </div> 
+       <div class="grid-2">
+        <img src="./images/Snapshoot Portfolio (1).png" alt="project2" class="project2">
+        <h1 class="ton">Tonic</h1>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 3]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+                   <li>HTML</li>
+                   <li>CSS</li>
+                   <li>JAVASCRIPT</li>
+               </ul>
+           <button class="see-project">See Project</button>
+      </div> 
+
+      <div class="grid-3">
+        <img src="./images/Snapshoot Portfolio (2).png" alt="project3" class="project3">
+        <h1 class="ton">Tonic</h1>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 4]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+                   <li>HTML</li>
+                   <li>CSS</li>
+                   <li>JAVASCRIPT</li>
+               </ul>
+           <button class="see-project">See Project</button>
+      </div>
+    
+      <div class="grid-4">
+        <img src="./images/Snapshoot Portfolio (3).png" alt="project4" class="project4">
+        <h1 class="ton">Multi-Post<br/> Stories</h1>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 5]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+                   <li>CSS</li>
+                   <li>JAVASCRIPT</li>
+               </ul>
+           <button class="see-project">See Project</button>
+      </div>  
+    </section>
+
+    <section class="about-section">
+        <div class="content-1">
+            <h1 class="ton">About Myself</h1>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure you have only one h1 tag in your HTML for the main title
for others, title and subtitle use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. :+1: 
   * This way will be semantically correct and the screen readers and other technologies will thank you.

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @manzitresor 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- Kindly make sure the cards are centered in the middle of the page. The margin left and right are the same which will make your design stick more to the figma design :+1: 

## Your design 

image

## Figma design 

image

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL]kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 6]
```diff
@@ -28,5 +29,163 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+               <span>
+                   <img src="./images/Counter.png">
+                   back end dev
+               </span>
+               <span>
+                   <img src="./images/Counter.png">
+                   2015
+               </span>
+           </div>
+           <p class="bio">
```
Comment (ghost) [inline index.html:72]: - [ ] Great work working on some of the changes. However, Your cards don't seem to be properly centered. They are aligned more to the right. Kindly adjust them to be aligned at the center to improve the general aesthetic of your portfolio

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @manzitresor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @manzitresor, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**